modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -13,6 +13,11 @@ Run cluster monitoring with persistent storage to gain the following benefits:
 
 For production environments, it is highly recommended to configure persistent storage. 
 
+[IMPORTANT]
+====
+In multi-node clusters, you must configure persistent storage for Prometheus, Alertmanager, and Thanos Ruler to ensure high availibility.
```

Comment:
it's not for HA, used PV to prevent data loss

Reply:
Hello @juzhao , thank you! I know it is to prevent data loss, but based on the conversation with Simon: "to be truly HA, all stateful components need to be configured with persistent storage". So I phrased it like this in this context. 

But I suppose that could be misleading? 
cc: @simonpasquier

Reply:
I don't think that it's misleading. Without persistent storage, we can't really claim that the stack is HA.

Reply:
availibility -> availability

Reply:
Oh yes thanks, I copied the text and did not notice the typo :grinning:  And thank you for clarifying the HA claim!

Reply:
@juzhao please let me know what you think :slightly_smiling_face: Thank you!